app/src/main/java/org/schabi/newpipe/MainActivity.java

```diff
@@ -131,10 +133,9 @@ protected void onCreate(final Bundle savedInstanceState) {
                     + "savedInstanceState = [" + savedInstanceState + "]");
         }
 
-        // enable TLS1.1/1.2 for kitkat devices, to fix download and play for media.ccc.de sources
-        if (Build.VERSION.SDK_INT == Build.VERSION_CODES.KITKAT) {
-            TLSSocketFactoryCompat.setAsDefault();
-        }
+        // use reasonable TLS settings (disable insecure, enable secure options)
+        HttpsURLConnection.setDefaultSSLSocketFactory(
+                NetCipher.getTlsOnlySocketFactory(false));
```

Comment:
```suggestion
                NetCipher.getTlsOnlySocketFactory());
```
getTlsOnlySocketFactory() internally calls getTlsOnlySocketFactory(false)